### DISCUSSION TITLE
关于我 | Alex Lin's Notes

### DISCUSSION THREAD

**Discussion by giscus** [chaoskyme/chaoskyme.github.io#321] [Comment]
# 关于我 | Alex Lin's Notes

嗨，我是 Alex，iOS 开发者，Web3 建设者。 热爱开源、分享与交流，欢迎关注我的GitHub。 技术栈： Native: iOS（ObjC、Swift）、Flutter、Rust Script: Python、Ruby、JavaScript Web3: Solidity、Ethereum、ethers.js、Wallet Dev、ERC-721、ERC-20

https://chaosky.tech/about/

**Comment by xwal** [comment]
测试新评论系统Giscus

**Comment by GuldanChen** [comment]
收藏了你的博客，哈哈哈，希望能交个朋友呢。

**Reply by xwal** [reply]
哈哈，感谢大佬，一起学习交流。

**Comment by axiaoxin** [comment]
路过
